src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString === '') {
+    return {};
+  }
+
+  const styleArray = sourceString.split(';');
+  const obj = {};
+
+  styleArray.forEach((declaration) => {
+    const parts = declaration.split(':');
+
+    if (parts.length === 2) {
+      const key = parts[0].trim();
+      const value = parts[1].trim();
+
+      obj[key] = value;
+    }
+  });
```

Comment:
This function does not handle the case where CSS properties contain ':' in their values (e.g., in a URL or a data URI). Consider using a regular expression or a more robust parsing strategy to correctly handle such cases.